q2_qemistree/plugin_setup.py

```diff
@@ -162,7 +163,9 @@
                         'ms2_matches': 'one or more tables with MS/MS library '
                                        'match for mass-spec features'},
     parameter_descriptions={'qc_properties': 'filters molecular properties to '
-                                             'retain PUBCHEM fingerprints'},
+                                             'retain PUBCHEM fingerprints',
+                            'metric' : 'metric for hierarchical clustering of '
+                                       'fingerprints'},
```

Comment:
I would suggest something like this to be explicit about the internal transformation:

```suggestion
                                       'fingerprints. If the Jaccard metric is'
                                       ' selected, molecular fingerprints are '
                                       'first binarized (probabilities above '
                                       '0.5 are True, and False otherwise).'},
```

Reply:
Thanks for suggesting that - I've added the new lines to the documentation. 
`zodiac_threshold` is used filter fingerprints with low-quality molecular formula predictions i.e. below the default score of 0.98 (now). I increased this value to be more stringent based on LF's benchmarking and the Zodiac paper pre-print.

`num_candidates` is the total number of fragmentation trees considered during zodiac reranking. A lower value makes the processing faster. Out previous default (75) was probably overkilling so I reduced it to 50 to speed things up.